hugo/layouts/shortcodes/code-tabs.html

```diff
@@ -6,7 +6,7 @@
 {{- $tabsBottomLeft := .Scratch.Get "tabs-bottom-left" -}}
 {{- $tabsBottomRight := .Scratch.Get "tabs-bottom-right" -}}
 
-<div class="code-tabs">
+<div class="code-tabs" data-copy>
```

Comment:
you want to copy the content of the current tab so I think data-copy should be on a nested element, probably the code block itself?